libs/navigation/footer.js

```diff
@@ -0,0 +1,25 @@
+const blockConfig = {
+  name: 'global-footer',
+  targetEl: 'footer',
+  appendType: 'appendChild',
+};
+
+async function loadBlock() {
+  const { fedsGlobalConfig = {} } = window;
+  const miloLibs = fedsGlobalConfig.miloLibs || 'https://stage--milo--adobecom.hlx.page';
```

Comment:
do we want STAGE hlx to be the default value?

Reply:
So, this will be main branch actually `https://main--milo--adobecom.hlx.page` and we had a plan to maintain another branch call feds in which the url would be `https://feds-milo--adobecom.hlx.page` which would kind of act like a stage environment for the consumers. 
Since Milo's stage branch gets updated frequently, consuming client wouldn't have enough time to test whenever there is any change in the code.
cc: @salonijain3 @amitbikram

Reply:
I think the default branch should be a stable one, also it would need to be something like www.adobe.com to avoid asking teams to allowlist adobecom.hlx.page / adobecom.hlx.live

Reply:
This is a very good point. I think we should point to a default stable branch and most likely www.adobe.com

Reply:
Updated to `https://www.adobe.com`
@narcis-radu @amitbikram

Reply:
That's also the fallback we have for normal federal content, adobe.com should always be accessible